drivers/mssqql.go

```diff
@@ -10,6 +10,7 @@ import (
 	_ "github.com/microsoft/go-mssqldb"
 	"github.com/xo/dburl"
 
+	"github.com/google/uuid"
```

Comment:
Please fix the imports order

```suggestion
	"github.com/google/uuid
	// mssql driver
 	_ "github.com/microsoft/go-mssqldb"
	"github.com/xo/dburl""
```